src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -132,56 +131,79 @@ public Response getRtFeeds() {
     private static HashMap<String, ScheduledExecutorService> runningTasks = new HashMap<>();
 
     @PUT
-    @Path("/{id}/{updateInterval}/monitor")
-    public Response getID(@PathParam("id") int id, @PathParam("updateInterval") int updateInterval) {
+    @Path("/{id}/{clientId}/{updateInterval}/monitor")
```

Comment:
Could you make these query string URL parameters instead of path parameters? This really applies to all the API calls. A variable should really only be a path  parameter if we are performing an operation on a particular object, and the path references the ID. Github uses this structure to refer to specific issues and pull requests. Things like update intervals and start and end times should really be query parameters, because their order in the path is somewhat arbitrary - they don't fit neatly into a hierarchy.